src/olympia/addons/models.py

```diff
@@ -1404,77 +1432,6 @@ def get_latest_file(self):
     def in_escalation_queue(self):
         return self.escalationqueue_set.exists()
 
-    def update_names(self, new_names):
```

Comment:
This was unused so I thought I can just remove it while I'm doing translation stuff.